ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -287,6 +287,10 @@ defaults <- node_group <- node_group.node <- node
     # MLAG interfaces (list) | Required when MLAG leafs present in topology.
     mlag_interfaces: [ < ethernet_interface_3 >, < ethernet_interface_4 > ]
 
+    # MLAG interfaces speed | Optional and depends on mlag_interfaces to be defined
+    # Can be defined at multiple levels -> defaults or node_groups or nodes
```

Comment:
This line can be removed. Covered in line 105.

Reply:
ok